scripts/locust/service.py

```diff
@@ -40,120 +81,251 @@ def parallel_requests():
     group.join()
 
 
-class ServiceTaskSet(TaskSet):
-    """This is the default page so start here"""
+# These task-less parent classes define common methods involving the locust HTTP
+# client.
```

Comment:
```suggestion
```

"Locust" is a proper noun. That a base class contains functionality common to all subclasses is implied.